### PR TITLE
Create address generator

### PR DESCRIPTION
**Pull request by piercebb**
## Changes
- Add `address` generator to `fake-eggs` library
- Add test checking for non-null and validate alphanumeric chars & spaces

### CODE REVIEW

File: src/generators/address/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import {Chance} from 'chance';
+
+import createAddressGenerator from '.';
+
+describe('address', () => {
+  it('generates a random address string', (): void => {
+    const chance = new Chance();
+    const address = createAddressGenerator(chance);
+
+    const addresValue = address();
```
Comment (jason-h-hu) [inline src/generators/address/index.test.ts:10]: typo: should be `addressValue`

**Review by jason-h-hu — APPROVED** [review]

**Review by serhalp — APPROVED** [review]
🎉

**Comment by serhalp** [comment]
funky Travis check is a holdover from a migration from travis-ci.org to travis-ci.com, so force-merging

**Comment by serhalp** [comment]
!merge
